content/news/2023-05-08-tpv-switch/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Job router switch on EU"
+date: "2023-05-08"
+tease: "Mothballing the Sorting Hat – how we migrated our dynamic job routing"
+hide_tease: false
+authors: 'Mira Kuntz, Helena Rasche'
+authors_structured:
+- github: mira-miracoli
+- github: hexylena
+tags: [EU]
+subsites: [all-eu]
+main_subsite: eu
+---
+
+
+# Mothballing the Sorting Hat
+
+## An old hat
+usegalaxy.eu runs more than 1 mio jobs per month. All these jobs have different demands and use different resources and technologies, like Docker, Apptainer etc., which need all sorts of parameters and environment variables. Galaxy handles jobs with resource managers like SLURM, HTCondor or the remote job execution system Pulsar.
+But how do these resource managers know which resources each individual job needs?  
+You could now specify that for every single tool, but as you can guess, with over 8.000 tools and many different possibilities to compute a job, this approach leads to chaotic unmaintainable static lists.
```

Comment:
I assume this should be localised differently?
```suggestion
You could now specify that for every single tool, but as you can guess, with over 8,000 tools and many different possibilities to compute a job, this approach leads to chaotic unmaintainable static lists.
```

Reply:
ah yes, I wrote it the German way